files/build_templates/sonic_debian_extension.j2

```diff
@@ -1128,6 +1128,10 @@ SONIC_PLATFORM_PY3_WHEEL_NAME=$(basename {{platform_api_py3_wheel_path}})
 sudo cp {{platform_api_py3_wheel_path}} $FILESYSTEM_ROOT/$SONIC_PLATFORM_PY3_WHEEL_NAME
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install $SONIC_PLATFORM_PY3_WHEEL_NAME
 sudo rm -rf $FILESYSTEM_ROOT/$SONIC_PLATFORM_PY3_WHEEL_NAME
+
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/libdashapi_*.deb || \
```

Comment:
If this is sonic-utilities's dependency, you do not need to install it explicitly.

Reply:
Yes it is sonic-utilities dependency, but the libdashapi debian package is not implicitly installed by specifying just the sonic-utilities dependency, 
```
admin@r-ocelot-07:~$ dump state --show
Traceback (most recent call last):
  File "/usr/local/bin/dump", line 5, in <module>
    from dump.main import dump
  File "/usr/local/lib/python3.11/dist-packages/dump/main.py", line 9, in <module>
    from dump.match_infra import RedisSource, JsonSource, MatchEngine, CONN
  File "/usr/local/lib/python3.11/dist-packages/dump/match_infra.py", line 10, in <module>
    from dump.dash_util import get_decoded_value
  File "/usr/local/lib/python3.11/dist-packages/dump/dash_util.py", line 6, in <module>
    from dash_api.types_pb2 import Guid, IpAddress, IpPrefix
ModuleNotFoundError: No module named 'dash_api'
```

Reply:
Then something is missing to achieve implicitly install of dependency. Could you check why?

Reply:
I think the `DEBS_DEPENDS` packages are not automatically installled in the target image, for example:
[In `sonic-yang-models-py3.mk` the py3 whl is dependent on `LIBYANG`](https://github.com/sonic-net/sonic-buildimage/blob/665c8e6f0e1ddffd4999954ccf77a0747edd8254/rules/sonic-yang-models-py3.mk#L4). But this [deb package is also explicitly installed along with the .whl in sonic_debian_extension.j2](https://github.com/sonic-net/sonic-buildimage/blob/665c8e6f0e1ddffd4999954ccf77a0747edd8254/files/build_templates/sonic_debian_extension.j2#L164)

Reply:
@qiluo-msft Created a new issue as previously discussed (The .deb is not installed automatically when *.whl is installed) https://github.com/sonic-net/sonic-buildimage/issues/20742 regarding the issue specified